libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -249,6 +257,8 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
             this.navigation = false;
         }
```

Comment:
Could you move this statement to line 242? I see you check for slides in both places.

Reply:
done